src/Files.App/Views/MainPage.xaml

```diff
@@ -119,67 +119,50 @@
 
 	<Grid>
 		<Grid.RowDefinitions>
+			<RowDefinition Height="Auto" />
```

Comment:
Is this intentional?

Reply:
Yes, this contains refactoring a bit.
Tab bar and address toolbar were wrapped with an unnecessary Grid control.